tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -209,6 +209,43 @@ public function testControllerInstanceForPrefixedRequest()
         Configure::write('App.namespace', $namespace);
     }
 
+    /**
+     * testTemplatePath
+     *
+     * @return void
+     */
+    public function testTemplatePath()
+    {
+        $request = (new ServerRequest())
+            ->withParam('controller', 'Foo')
+            ->withParam('action', 'bar');
+        $exception = new NotFoundException();
+        $ExceptionRenderer = new ExceptionRenderer($exception, $request);
+
+        $ExceptionRenderer->render();
+        $controller = $ExceptionRenderer->__debugInfo()['controller'];
+        $this->assertEquals('error400', $controller->viewBuilder()->getTemplate());
+        $this->assertEquals('Error', $controller->viewBuilder()->getTemplatePath());
+
+        $request = $request->withParam('prefix', 'admin');
+        $exception = new MissingActionException(['controller' => 'Foo', 'action' => 'bar']);
+
+        $ExceptionRenderer = new ExceptionRenderer($exception, $request);
+
+        $ExceptionRenderer->render();
+        $controller = $ExceptionRenderer->__debugInfo()['controller'];
+        $this->assertEquals('missingAction', $controller->viewBuilder()->getTemplate());
+        $this->assertEquals('Error', $controller->viewBuilder()->getTemplatePath());
+
+        Configure::write('debug', false);
+        $ExceptionRenderer = new ExceptionRenderer($exception, $request);
+
+        $ExceptionRenderer->render();
+        $controller = $ExceptionRenderer->__debugInfo()['controller'];
+        $this->assertEquals('error400', $controller->viewBuilder()->getTemplate());
+        $this->assertEquals('Admin/Error', $controller->viewBuilder()->getTemplatePath());
```

Comment:
This assertion is failing on AppVeyor because DIRECTORY_SEPARATOR is different there.